### ISSUE TITLE
[Bug] Being able to lock pick outside of a car without needing to do multiple attempts

### ISSUE THREAD

**Issue by Twistedbitt** [FrenzyRoleplay/server-issues#623]
**Describe the bug**
When you are lockpicking a car, if a local or yourself leave the car, and still have the lockpicking mini game on your screen and you re-enter the car, you will always get the car on the first bar, The other 1 or 2 bars do not show up.

**To Reproduce**
1. Go to any car
2. Start to lockpick the car
3. press F to leave the car and then re-enter it
4. Finish the first bar of the mini game and you now have a car fully lockpicked without needing to do the mini games 2-3 times as its intended.

**Expected behavior**
Not being able to leave the car  or the lockpicking to stop if you exit the car or pulled out while lockpicking

**Additional context**
Basically, If i am to lockpick a car, press F while the mini game is up and F again to re-enter the car and beat the first bar of the mini game, the car is yours and lockpicked. It wont move on to any other stage of the lock picking. This can be abused a lot.
